core/lib/via_btc_client/src/indexer/mod.rs

```diff
@@ -204,28 +229,28 @@ impl BitcoinInscriptionIndexer {
     fn is_valid_message(&self, message: &FullInscriptionMessage) -> bool {
         match message {
             FullInscriptionMessage::ProposeSequencer(m) => {
-                let is_valid = Self::get_sender_address(&m.common, self.network)
-                    .map_or(false, |addr| self.verifier_addresses.contains(&addr));
-                debug!("ProposeSequencer message validity: {}", is_valid);
-                is_valid
+                // let is_valid = parser::get_btc_address(&m.common, self.network)
```

Comment:
@steph-mipt Let's create tasks for this part

and fix it as we discussed in slack ( getting the p2wpkh address from fee payer utxo sender instead of compress public key inside of inscription message.